website_field_autocomplete/controllers/main.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import json
+from odoo import http
+from odoo.http import request
+from odoo.addons.website.controllers.main import Website
+
+
+class Website(Website):
+
+    @http.route(
+        '/website/field_autocomplete/<string:model>',
+        type='http',
+        auth='public',
+        methods=['POST'],
+        website=True,
+    )
+    def _get_field_autocomplete(self, model, **kwargs):
```

Comment:
routes' methods shouldn't be public methods?

Reply:
They don't have to be as far as I'm aware. I see no need to change this as part of a migration, since I'm not the original author I don't know if it was a conscious choice or not...